database/connect.sh

```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
```

Comment:
Nice way to save having to enter the password when connecting

Reply:
Saves time, otherwise the user would manually have to enter the password each time. LGTM.